demo/src/pick_place_task.cpp

```diff
@@ -153,7 +156,8 @@ void PickPlaceTask::init() {
 	 *               Move to Pick                       *
 	 *                                                  *
 	 ***************************************************/
-	{  // Move-to pre-grasp
+	// Move to a pre-grasp pose. The Connect stage does not declare any movements but connects two stages that do.
```

Comment:
```suggestion
	// Move to the pre-grasp pose. The Connect stage connects two other stages with a motion.
```
Not perfectly happy with this, but it sounded too much like there's no robot motion going on in this stage